components/mosquitto/examples/serverless_mqtt/README.md

```diff
@@ -0,0 +1,53 @@
+# Serverless MQTT Example
+
+MQTT served by (two) mosquitto's running on two ESP chips.
+
+* Leverages MQTT connectivity between private two networks without cloud premisses.
+* Creates two local MQTT servers (on ESP32x's) which are being synchronized over peer to peer connection (established via ICE protocol, by libjuice).
```

Comment:
Would be good to provide link ICE protocol, by libjuice

Reply:
will add the link here, as well

Reply:
added link